crates/knope-versioning/src/versioned_file/mod.rs

```diff
@@ -35,6 +37,10 @@ pub enum VersionedFile {
     PackageJson(PackageJson),
     PyProject(PyProject),
     MavenPom(MavenPom),
+    TauriConf(TauriConfJson),
+    TauriMacosConf(TauriConfJson),
+    TauriWindowsConf(TauriConfJson),
+    TauriLinuxConf(TauriConfJson),
```

Comment:
I see what you mean about the 1-1 file type to path being problematic. We should definitely decouple those and let there be multiple file names per type. I'll take a stab at that